palm/plugins/core/commands/cmd_override.py

```diff
@@ -0,0 +1,28 @@
+import click
+import shutil
+from pathlib import Path
+
+
+@click.command('override')
+@click.option('--name', multiple=False, required=True, help='Name of the command')
+@click.pass_obj
+def cli(environment, name: str):
+    """Override a command in the current project"""
+    spec = environment.plugin_manager.command_spec(name)
+    origin_path = Path(spec.origin)
+    file_name = origin_path.name
+    target_path = Path(environment.palm.project_root, '.palm', file_name)
+
+    if target_path.exists():
+        click.secho('Command already exists in project, skipping', fg='red')
```

Comment:
will we ever want to override the override with this command?

Reply:
If it already exists in your repo, then you'd just edit the file. There's nothing really to override, you already overrode it!